release-notes/release-notes522.md

```diff
@@ -32,10 +38,16 @@ The following premium plugins updates were released alongside {{site.productname
 
 ### PowerPaste 5.2.2
 
-The {{site.productname}} 5.2.3 release includes an accompanying release of the **PowerPaste** premium plugin.
+The {{site.productname}} 5.2.2 release includes an accompanying release of the **PowerPaste** premium plugin.
```

Comment:
would this release bump powerpaste to 5.2.4 ?

I believe its currently 5.2.3
https://github.com/tinymce/tinymce-enterprise/blob/develop/artifacts.js

Reply:
That has already been bumped to the version being released.

Note: I also edited the above as that was linking to a private repo in a public PR.